### DISCUSSION TITLE
hyper parameter documentation and range

### DISCUSSION THREAD

**Discussion by prabhant** [online-ml/river#561] [Q&A] [answered]
Hi is there any documentation regarding the range of hyperparameters in classifiers
For example:
https://riverml.xyz/dev/api/linear-model/ALMAClassifier/

How can I find the accepted values of parameters in this classifier?

**Comment by MaxHalford** [comment, marked as answer]
There isn't any! This should be documented though. If it isn't we should. We could also imagine using type hints or what not. But I think documentation is the way to go.

**Reply by prabhant** [reply]
Thanks, that will be really helpful

**Reply by MaxHalford** [reply]
If you find a hyperparameter that is not well documented, please consider sending a pull request :). Even if the pull request is not 100% correct, we can fix it together.
